bin/pa11y

```diff
@@ -70,6 +70,10 @@ function configureProgram(program) {
 			'the time to wait before running tests in milliseconds'
 		)
 		.option(
+			'-j, --javascript <script>',
```

Comment:
I'm not sure on the naming of this option so just went with this for now, better suggestions are very welcome.